kitsune/wiki/tests/test_parser.py

```diff
@@ -840,7 +828,7 @@ def test_links(self):
         eq_([d.linked_from.title for d in d2.links_to()], ['D3'])
 
     def test_templates(self):
-        d1, _, _ = doc_rev_parser('Oh hai', title='Template:D1')
+        d1, _, _ = doc_rev_parser('Oh hai', title='Template:D1', category=TEMPLATES_CATEGORY)
         d2, _, _ = doc_rev_parser('[[Template:D1]]', title='D2')
```

Comment:
Probably worth using the `TEMPLATE_PREFIX_STRING` here, too.

I'll stop marking this. Point being that we've got a constant so we should use it to make life easier and reduce likelihood of spelling mistakes and all that.